c2corg_ui/static/js/viewdetails.js

```diff
@@ -334,10 +334,16 @@ app.ViewDetailsController.prototype.createTopic = function() {
   var lang = document.lang;
   this.api_.createTopic(document_id, lang).then(function(resp) {
     var topic_id = resp['data']['topic_id'];
-    this.documentService.document.topic_id = topic_id;
-    this.getComments();
     var url = this.discourseUrl_ + 't/' + document_id + '_' + lang + '/' + topic_id;
     window.location = url;
+  }.bind(this), function(resp) {
+    if (resp.status == 400) {
+      var topic_id = resp.data.errors[0].topic_id;
```

Comment:
To avoid problems in compiled mode, better use `resp.data.errors[0]['topic_id']` here if you don't want to create an externs type.